CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.8 (2021-07-26)
+
+- Fixed the sectioned form component to have proper spacing between a section's heading and invalid icon. [#176](https://github.com/blackbaud/skyux-tabs/pull/176)
```

Comment:
```suggestion
- Fixed the sectioned form component to have proper spacing between a section's heading and an invalid icon. [#176](https://github.com/blackbaud/skyux-tabs/pull/176)
```

Reply:
@blackbaud-paulah - I'm not sure if the `an` feels right to me here. The `an` to me makes it sound like there can just be any generic invalid icon; however, this one is baked into the section. So both the heading and icon are things that the "section" is possessive of. There can only ever be one invalid icon on a section and it can not be modified - just turned on and off. Thoughts?

Reply:
Thanks for explaining, Trevor. That makes a lot of sense. In that case, I will delete my suggestion. :)